packages/x-components/src/views/home/Home.vue

```diff
@@ -126,7 +134,7 @@
           >
             <span>{{ $x.totalResults }} Results</span>
             <BaseColumnPickerList
-              #default="{ column }"
+              v-slot="{ column }"
```

Comment:
I guess it was auto-fixed by eslint, we should keep with `#default`.
```suggestion
              #default="{ column }"
```